encrypted-media/Google/encrypted-media-lifetime-reload.html

```diff
@@ -86,7 +86,7 @@
                 }).then(function(mediaKeys) {
                     waitForEventAndRunStep('encrypted', video, onEncrypted, test);
                     waitForEventAndRunStep('playing', video, onPlaying, test);
-                    video.src = '../content/test-encrypted.webm';
+                    video.src = 'content/test-encrypted.webm';
```

Comment:
FWIW, MSE tests have each type in a [separate directory](https://github.com/w3c/web-platform-tests/tree/master/media-source/webm). I think this is fine, and probably scales better to the various combinations. @wolenetz